### PR TITLE
FEATURE: add hostuser and clone command

### PR DESCRIPTION
**Pull request by freesh**
- On _make install_ an additional hostuser will be created on the php-fpm container .
- A new shiny command is entering the stage: _make clone_

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -167,6 +173,14 @@ ssh-mariadb::
 ssh-webserver::
 	docker-compose exec -w /etc/nginx webserver sh
 
+###############################################################################
+#                                CLONE                                        #
+###############################################################################
+clone::
+	@docker-compose exec --user $(HOST_USER) php-fpm ssh-agent /bin/bash -c "\
+		./flow clone:list; \
+		./flow clone:preset --yes"
```
Comment (grebaldi) [inline Makefile]: It would be great to be able to pass a preset via `make clone preset=dev`.

File: Makefile [hunk 2]
```diff
@@ -64,9 +64,15 @@ environment::
...
 
+@install-create-user::
+	@$(COMPOSE_EXEC_ROOT) mkdir -p /home/hostuser
+	@$(COMPOSE_EXEC_ROOT) useradd -u $(HOST_USER) hostuser
+	@$(COMPOSE_EXEC_ROOT) chown hostuser /home/hostuser
+
 install::
 	$(MAKE) -s up
 	@time $(MAKE) -s -j 3 @install-githooks @install-composer @install-yarn
+	$(MAKE) -si @install-create-user
```
Comment (grebaldi) [inline Makefile]: Couldn't this actually run in parellel with the others?

Comment (freesh) [reply Makefile]: Good point. I moved it to the make up command und run it in parallel with the chmod command for /data

File: Makefile (hunk 1 above)
Comment (freesh) [reply Makefile]: I don't get it running. Maybe we should take a look together.

Comment (freesh) [reply Makefile]: done. :)

**Pull request merged**